Example Apps/Example ObjC/RemoteControlManager.m

```diff
@@ -0,0 +1,248 @@
+//
+//  RemoteControlManager.m
+//  SmartDeviceLink-Example-ObjC
+//
+//  Created by Beharry, Justin (J.S.) on 8/1/22.
+//  Copyright © 2022 smartdevicelink. All rights reserved.
+//
+
+#import "RemoteControlManager.h"
+
+#import "AlertManager.h"
+#import "SmartDeviceLink.h"
+
+@interface RemoteControlManager()
+
+@property (strong, nonatomic) SDLManager *sdlManager;
+@property (assign, nonatomic, getter=isPermissionEnabled, readwrite) BOOL permissionEnabled;
```

Comment:
```suggestion
```

This doesn't need to be here if the only access is by `_permissionEnabled` in the init